### PR TITLE
Reduce noise in logs

### PR DESCRIPTION
**Pull request by lekkas**
[Review on Reviewable](https://reviewable.io/reviews/resin-io/resin-supervisor/102)

connects to #101

### CODE REVIEW

**Comment by lekkas** [comment]
@pcarranzav Do you think we should also consider removing [this](https://github.com/resin-io/resin-supervisor/commit/b550708e983f8f91efea44d4d05e2da6d0ada896#diff-e616ed67248b05ac46c3606a0781fb9cL195) ? I have no easy way of checking if this is also polluting supervisor logs in real devices.

**Comment by pcarranzav** [comment]
@lekkas yes, we should remove that one too

File: src/docker-utils.coffee [hunk 1]
```diff
@@ -204,8 +204,7 @@ do ->
 						docker.getImage(tag).removeAsync()
 						.then ->
 							console.log('Deleted image:', tag, image.Id, image.RepoTags)
-						.catch (err) ->
-							console.log('Error deleting image:', tag, image.Id, image.RepoTags, err)
+						.catch (err) -> # Ignore image delete error
```
Comment (pcarranzav) [inline src/docker-utils.coffee]: No need for `err` here.
Also might look nicer by defining:
```
ignore  = ->
```
somewhere above,and then here (and in the other error to ignore) do this:
```
.catch(ignore)
```

Comment (lekkas) [inline src/docker-utils.coffee]: I'd choose the empty `catch ->` version over defining a new `ignore` function, which I find a bit verbose. What do you think?

Comment (pcarranzav) [inline src/docker-utils.coffee]: I just feel it makes it easier to read ([example](https://github.com/resin-io/resin-wifi-connect/blob/master/src/app.coffee#L149)), but it's up to you.

**Comment by pcarranzav** [comment]
LGTM

File: src/docker-utils.coffee (hunk 1 above)
Comment (Page-) [inline src/docker-utils.coffee]: You can also use `_.noop` to be a bit more English-language, and reusing a single noop function helps the optimiser a bit as well

Comment (pcarranzav) [inline src/docker-utils.coffee]: +1 on using `_.noop`

**Comment by pcarranzav** [comment]
connects to #101

**Comment by pcarranzav** [comment]
@lekkas could you add a "connects to  #101" in the PR opening comment? That way waffle picks it up and associates the PR with the issue.

File: src/docker-utils.coffee (hunk 1 above)
Comment (lekkas) [inline src/docker-utils.coffee]: `_.noop` it is

**Comment by pcarranzav** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/resin-io/resin-supervisor/102)*

**Comment by pcarranzav** [comment]
@Page- we need your approval in the convo on reviewable please :)

**Comment by Page-** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/resin-io/resin-supervisor/102)*

**Pull request merged**